src/renderer/components/layout/PlayDetail/components/MusicComment/index.vue

```diff
@@ -1,7 +1,7 @@
 <template lang="pug">
 div.comment(ref="dom_container" :class="$style.comment")
   div(:class="$style.commentHeader")
-    h3 {{ $t('comment__title', { name: title }) }}
+    h3 {{ $t('comment__title', { name: currentMusicInfo.name }) }}
```

Comment:
Why change this? `title` is formatted

Reply:
Remove the following `title() { ...` and `import { appSetting } from '@renderer/store/setting'`

Reply:
And `  setup() {...`